etn-ccis/blui-react-workflows
# Feature/4128 login screen full

Fixes BLUI-4128.



#### Changes proposed in this Pull Request:

- add LoginScreen [full screen component]
- add unit tests for login screen




#### Screenshots / Screen Recording (if applicable)

![Screen Shot 2023-06-13 at 4 12 31 PM](https://github.com/etn-ccis/blui-react-workflows/assets/13989985/888590fb-9d19-4c2f-a34c-a097806e02a2)




#### To Test:

- `yarn start:example2`
- click the "Test Login Full Screen" button or go to http://localhost:3000/login-screen-full
- ensure the functionality matches the acceptance criteria in the issue description
- ensure `yarn test` passes all tests
